external-crates/move/crates/move-analyzer/src/analyzer.rs

```diff
@@ -44,7 +44,7 @@ pub fn run() {
     );
 
     let (connection, io_threads) = Connection::stdio();
-    let symbols = Arc::new(Mutex::new(symbols::empty_symbols()));
+    let symbols_map = Arc::new(Mutex::new(BTreeMap::new()));
```

Comment:
This change is not directly related to enums support but enums support would not work if we did not merge `CompilerInfo` at the end of symbols computation. When thinking about it, I realized that merging is not the greatest idea after all and figured I should fix it